scripts/jenkins/cloud/ansible/deploy-caasp-v4-terraform.yml

```diff
@@ -0,0 +1,568 @@
+
+- name: Deploy CaaSPv4 using terraform
+  hosts: managementmachine
+  gather_facts: False
+  remote_user: sles
+  vars:
+    ca_path: "/etc/pki/trust/anchors/ca-bundle.pem"
+    caasp_deployment_path: "/home/sles/caasp/deployment"
+    caasp_openstack_path: "{{ caasp_deployment_path }}/openstack"
+    cpiautotfvars_path: "{{ caasp_openstack_path }}/cpi.auto.tfvars"
+    dns_server_ip1: "10.84.2.20"
+    dns_server_ip2: "10.84.2.21"
+    external_net: "ext-net"
+    image_name: "SLE-15-SP1-JeOS-QU2"
+    internal_net: "caasp-int-net"
+    internal_subnet: "caasp-int-subnet"
+    lb_tf_path: "{{ caasp_openstack_path }}/load-balancer.tf"
+    management_machine_repos:
+      - name: sles-15-sp1
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE:/SLE-15-SP1:/GA/standard/
+      - name: sles-15-sp1-update
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE:/SLE-15-SP1:/Update/standard/
+      - name: base-system-pool
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/
+      - name: base-system-update
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/
+      - name: cassp40-pool
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE:/SLE-15-SP1:/Update:/Products:/CASP40/standard/
+      - name: caasp40-update
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE:/SLE-15-SP1:/Update:/Products:/CASP40:/Update/standard/
+    master_size: "m1.medium"
+    network_tf_path: "{{ caasp_openstack_path }}/network.tf"
+    #Due to resource constraints in virtual cloud environment setup, the num_masters and num_workers are set to 1
+    #This can be overidden for baremetal QE cloud environments.
+    num_masters: 1
+    num_workers: 1
+    registration_auto_vars_path: "{{ caasp_openstack_path }}/registration.auto.tfvars"
+    repositories: '\n
+      basesystem_pool = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/" \n
+      serverapps_pool = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Server-Applications/15-SP1/x86_64/product/" \n
+      containers_pool = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Containers/15-SP1/x86_64/product/" \n
+      basesystem_updates = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/" \n
+      containers_updates = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Containers/15-SP1/x86_64/update/" \n
+      serverapps_updates = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Server-Applications/15-SP1/x86_64/update/" \n
+    '
+    ssh_key_filename: "id_rsa"
+    stack_name: "auto-caasp"
+    subnet_cidr: "172.28.0.0/24"
+    terraform_destroy: false
+    tfvars_path: "{{ caasp_openstack_path }}/terraform.tfvars"
+    timestamp: ""
+    service_caasp_osrc: "service_caasp.osrc"
+    user: "sles"
+    worker_size: "m1.medium"
+  tasks:
+  - name: Use python3
```

Comment:
Do you really need this task? wouldn't setting this alongside the other playbooks variables work?